pkg/release/release.go

```diff
@@ -208,6 +208,7 @@ func (r *Release) Sync(client helm.Client, hr *v1.HelmRelease) (rHr *v1.HelmRele
 		ResetValues: hr.Spec.ResetValues,
 		MaxHistory:  hr.GetMaxHistory(),
 		Wait:        hr.Spec.Wait || (curRel != nil && hr.Spec.Rollback.Enable),
+		SkipCRDs:    hr.Spec.SkipCRDs,
```

Comment:
@hiddeco I know you are a stickler for ordering of the properties so let me know if you want it moved :smile:

Reply:
Haha, I'd probably have done something like this:

```go
...
Install:     curRel == nil,
Force:       hr.Spec.ForceUpgrade,
ResetValues: hr.Spec.ResetValues,
SkipCRDs:    hr.Spec.SkipCRDs,
MaxHistory:  hr.GetMaxHistory(),
...
```

Reply:
Done, reordered in the CRD too.